src/uct/ib/efa/configure.m4

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+#
+# See file LICENSE for terms.
+#
+
+#
+# EFA provider Support
+#
+AC_ARG_WITH([efa],
+            [AS_HELP_STRING([--with-efa], [Enable EFA device support (default is guess)])],
+            [], [with_efa=guess])
+
+have_efa=no
+
+AS_IF([test "x$with_ib" = "xno"], [with_efa=no])
```

Comment:
did you mean this?
```suggestion
AS_IF([test "x$with_ib" != "xno"], [with_efa=no])
```

Reply:
If we don't have the top level common IB module, we cannot have EFA module, so I still think line 16 is right.